packages/api-derive/src/contracts/fees.ts

```diff
@@ -47,12 +48,13 @@ function queryConstants (api: ApiInterfaceRx): Observable<ResultV2> {
 export function fees (api: ApiInterfaceRx): () => Observable<DeriveContractFees> {
   return memo((): Observable<DeriveContractFees> => {
     return queryConstants(api).pipe(
-      map(([creationFee, transferFee, callBaseFee, contractFee, rentByteFee, rentDepositOffset, tombstoneDeposit, transactionBaseFee, transactionByteFee]): DeriveContractFees => ({
+      map(([callBaseFee, contractFee, creationFee, rentByteFee, rentDepositOffset, surchargeReward, tombstoneDeposit, transactionBaseFee, transactionByteFee, transferFee]): DeriveContractFees => ({
```

Comment:
```suggestion
      map(([callBaseFee, contractFee, creationFee, transactionBaseFee, transactionByteFee, transferFee, rentByteFee, rentDepositOffset, surchargeReward, tombstoneDeposit]): DeriveContractFees => ({
```

This needs to match with the order above. the transaction* fees are out of order.

Reply:
My bad. I previously thought the args order need to be sorted alphabetically. Now, I surely rearrange the args in the right way :)